lib/main.dart

```diff
@@ -83,15 +83,24 @@ void main() async {
   try {
     await setupLogging();
     await _setupEdgeToEdgeOverlayStyle();
+    Logger("Main()").info("Setup edge-to-edge overlay");
     await setupHive();
+    Logger("Main()").info("Setup hive and isar");
     _migrateDownloadLocations();
     _migrateSortOptions();
+    Logger("Main()").info("Completed applicable migrations");
     await _setupFinampUserHelper();
+    Logger("Main()").info("Setup user helper");
     await _setupJellyfinApiData();
+    Logger("Main()").info("setup jellyfin api");
     _setupOfflineListenLogHelper();
+    Logger("Main()").info("Setup offline listen tracking");
     await _setupDownloadsHelper();
+    Logger("Main()").info("Setup downloads service");
     await _setupOSIntegration();
+    Logger("Main()").info("Setup os integrations");
     await _setupPlaybackServices();
+    Logger("Main()").info("Setup audio player");
```

Comment:
nice. using a variable instead of always getting the singleton based on the name might be a good idea though